### PR TITLE
Updates to channel database and channel status pages

### PR DESCRIPTION
**Pull request by tannerbk**
- Add GTValid information (database is currently empty)
- Add Vthr information to channel status page even when the detector is off, using the most recent run
- Update to channel database page to list the status similarly to the channel status page and to include a filter for pmt type.

### CODE REVIEW

File: minard/channeldb.py [hunk 1]
```diff
@@ -54,9 +54,9 @@ def pmt_type_description(pmt_type):
     elif pmt_type == 0x40:
         return "OWL"
     elif pmt_type == 0x80:
-        return "Butt"
+        return "BUTT"
     elif pmt_type == 0x12:
-        return "Petal-less PMT"
+        return "Petalless PMT"
```
Comment (tlatorre-uchicago) [inline minard/channeldb.py]: Why is this changed? I kind of liked Petal-less.

File: minard/channeldb.py [hunk 2]
```diff
@@ -85,7 +85,7 @@ def get_channels(kwargs, limit=100):
     query = "SELECT * FROM current_channel_status "
     if len(kwargs):
         query += "WHERE %s " % (" AND ".join(["%s = %%(%s)s" % (item[0], item[0]) for item in kwargs.items()]))
-    query += "ORDER BY crate, slot, channel LIMIT %i" % limit
+    query += "ORDER BY crate, slot, channel"
```
Comment (tlatorre-uchicago) [inline minard/channeldb.py]: Why did you remove the limit here?

File: minard/views.py [hunk 3]
```diff
@@ -740,7 +766,7 @@ def check_rates_history():
     starting_run = request.args.get('starting_run',103214,type=int)
 
     data, stats = polling_history(crate, slot, channel, starting_run)
-    discriminator_threshold = get_discriminator_threshold(crate, slot, channel)
+    discriminator_threshold = get_discriminator_threshold(crate, slot)
     return render_template('check_rates_history.html', crate=crate, slot=slot, channel=channel, data=data, stats=stats, discriminator_threshold=discriminator_threshold)
```
Comment (tlatorre-uchicago) [inline minard/views.py]: Is this not going to fail with the different return type? If so, maybe it's better to just create a new function get_discriminator_thresholds_for_slot or something.

File: minard/templates/channel_database.html [hunk 4]
```diff
@@ -158,54 +176,28 @@
             {% endif %}
             </td>
         {% endmacro %}
-	    <div class="col-md-12">
-		<table class="table table-hover text-center table-condensed">
+	    <div class="col-md-10">
```
Comment (tlatorre-uchicago) [inline minard/templates/channel_database.html]: I think it looks better with col-md-12. Is there a reason to change it to 10?

File: minard/views.py [hunk 5]
```diff
@@ -216,8 +216,33 @@ def detector_state_check(run=None):
...
+    pmt_info = get_pmt_types()
+
+    # Allow sorting by PMT Type and apply the limit
+    row_count = 0
+    new_results = []
+    for row in results:
+        crate = row['crate']
+        slot = row['slot']
+        channel = row['channel']
+        if pmt_type == -1:
```
Comment (tlatorre-uchicago) [inline minard/views.py]: What's up with this check?

File: minard/views.py [hunk 6]
```diff
@@ -216,8 +216,33 @@ def detector_state_check(run=None):
...
+        elif pmt_info[crate][slot][channel] != pmt_type and \
+             pmt_info[crate][slot][channel] != active_type:
+            continue
+        row_count+=1
+        status = filter_channel_status(row)
+        row['status'] = status
+        row['pmt_type'] = pmt_type_description(pmt_info[crate][slot][channel])
+        new_results.append(row)
+        if row_count >= limit:
+            break
```
Comment (tlatorre-uchicago) [inline minard/views.py]: This works, but seems a little hacky. I wonder if it would be better to add all this to the get_channels function by doing a join in the query, i.e.

```sql
SELECT * FROM current_channel_status, pmt_info WHERE current_channel_status.crate = pmt_info.crate AND current_channel_status.slot = pmt_info.slot AND current_channel_status.channel = pmt_info.channel
```

That way you should just be able to pass the type parameter to the get_channels function and it will all just work.

File: minard/views.py (hunk 3 above)
Comment (tannerbk) [reply minard/views.py]: The function never used the channel. I was just removing unused code.

File: minard/views.py (hunk 6 above)
Comment (tannerbk) [reply minard/views.py]: Thanks, that was quite helpful, the new implementation should be much less hacky.
